backend/app/services/llm/providers/gai.py

```
@@ -0,0 +1,136 @@
+                    completion_type=completion_type,
+                    provider=provider,
+                )
+
+                model = generation_params.get("model")
+                instructions = generation_params.get("instructions", "")
+                input_language = generation_params.get("input_language") or "auto"
+                output_language = generation_params.get("output_language")
+
+                # Build transcription/translation instruction
+                if input_language == "auto":
+                    lang_instruction = f"Detect the spoken language automatically and transcribe the audio"
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)